experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldToDelta.ts

```diff
@@ -74,9 +94,24 @@ function cellDeltaFromMark<TNodeChange>(
 				return mark.count;
 			}
 			case "Delete": {
```

Comment:
The Revive case (on which GH doesn't let me put a comment) also needs to be updated to produce the inverse move.

Reply:
This was discussed offline and will be done in a separate PR.